collect_app/src/main/java/org/odk/collect/android/activities/CaptureSelfieActivity.java

```diff
@@ -98,9 +118,23 @@ private Camera getCameraInstance() {
 
             if (camInfo.facing == Camera.CameraInfo.CAMERA_FACING_FRONT) {
                 camera = Camera.open(camNo);
+                cameraInfo = camInfo;
                 camera.setDisplayOrientation(90);
+
             }
         }
+
+        // Set the rotation of the camera which the output picture need.
+        if (camera != null) {
+            Camera.Parameters parameters;
+            parameters = camera.getParameters();
```

Comment:
These two lines can be combined: 
`Camera.Parameters parameters = camera.getParameters();`
If `parameters` were in scope a bit longer, I would recommend adding `final`.